ocaml/xapi/xapi_host.ml

```diff
@@ -916,6 +916,12 @@ let create ~__context ~uuid ~name_label ~name_description ~hostname ~address
   let metrics = Ref.make () in
   make_new_metrics_object metrics ;
   let host_is_us = uuid = Helpers.get_localhost_uuid () in
+  let tls_verification_enabled =
+    if host_is_us then
+      Stunnel_client.get_verify_by_default ()
+    else
+      false
```

Comment:
If I'm not mistaken this is use when a host joins a pool. In that case it should be set to the default of the pool instead of just `false` unconditionally.